Project/Sources/Classes/OAuth2Provider.4dm

```diff
@@ -794,3 +833,59 @@ Function get tokenURI() : Text
 	End case 
 	
 	return $tokenURI
+	
+	
+Function get hexToBase64Url() : Text
```

Comment:
is needed to be public? (ie. without underscore _ )

Reply:
IMO this function is unnecessary. Calling :

`
BASE64 ENCODE(_UrlEncode(This._thumbprint);$result)
`
Will do the job.